dart/test/sentry_envelope_vm_test.dart

```diff
@@ -50,248 +53,21 @@ void main() {
       final sdkVersion = SdkVersion(name: '', version: '');
       final attachment =
           IoSentryAttachment.fromPath('this_path_does_not_exist.txt');
-
       final envelope = SentryEnvelope.fromEvent(
         event,
         sdkVersion,
+        dsn: fakeDsn,
         attachments: [attachment],
       );
 
       final data = (await envelope.envelopeStream(SentryOptions()).toList())
           .reduce((a, b) => a + b);
 
-      expect(data, envelopeBinaryData);
+      final file = File('test_resources/envelope-no-attachment.envelope');
+      final jsonStr = await file.readAsString();
+      final dataStr = utf8.decode(data);
+
+      expect(dataStr, jsonStr);
```

Comment:
For some reason, this fails on Windows:

> Expected: ... 1234567"}\r\n{"conte ...
> Actual: ... 1234567"}\n{"content ...
>                                     ^
> Differ at offset 128

Comparing the bytes directly also fail, only on Windows.

Reply:
> some reason

> \r\n

> \n

Reply:
@ueman @vaind ideas why this is failing?
Don't wanna lose more time on this =D

Reply:
At some places we do `utf8.encode('\n')`
Maybe we don't need to do that? or maybe this has different behavior on Windows?

Reply:
> The
> 
> > some reason
> 
> seems to be the different new line characters:
> 
> > \r\n
> 
> > \n

yeah I know, but the question is how the `\r` gets added in there only on Windows.

Reply:
> At some places we do utf8.encode('\n')

Because the Sentry spec demands it.

> Maybe we don't need to do that? or maybe this has different behavior on Windows?

I'm not aware of such behavior.

One solution could be 
https://github.com/getsentry/sentry-dart/pull/1050/files#r990065322

Reply:
see https://github.com/actions/checkout/issues/135

Reply:
And its green, thanks a lot @vaind

Reply:
my pleasure